circleci/docker-publish

```diff
@@ -86,9 +88,13 @@ docker build -t $ORG/$REPO:$SHORT_SHA .
 
 # ECR login.
 echo "Logging into ECR..."
-ecr_login $ECR_REGION_PRIMARY $ECR_PUSH_ID $ECR_PUSH_SECRET
-ecr_login $ECR_REGION_SECONDARY $ECR_PUSH_ID $ECR_PUSH_SECRET
+ecr_login $ECR_REGION_US_WEST_1 $ECR_PUSH_ID $ECR_PUSH_SECRET
+ecr_login $ECR_REGION_US_WEST_2 $ECR_PUSH_ID $ECR_PUSH_SECRET
+ecr_login $ECR_REGION_US_EAST_1 $ECR_PUSH_ID $ECR_PUSH_SECRET
+ecr_login $ECR_REGION_US_EAST_2 $ECR_PUSH_ID $ECR_PUSH_SECRET
 
 echo "Pushing to ECR..."
-push_ecr_image $ECR_REGION_PRIMARY
-push_ecr_image $ECR_REGION_SECONDARY
+push_ecr_image $ECR_REGION_US_WEST_1
+push_ecr_image $ECR_REGION_US_WEST_2
+push_ecr_image $ECR_REGION_US_EAST_1
+push_ecr_image $ECR_REGION_US_EAST_2
```

Comment:
how stateful is the call to `ecr_login`? Wondering if these need to be sequenced as pairs, e.g. 

```
ecr_login $ECR_REGION_US_WEST_1 $ECR_PUSH_ID $ECR_PUSH_SECRET
push_ecr_image $ECR_REGION_US_WEST_1

ecr_login $ECR_REGION_US_WEST_2 $ECR_PUSH_ID $ECR_PUSH_SECRET
push_ecr_image $ECR_REGION_US_WEST_2
```
etc.

Reply:
Thanks for the testing tip! I'll try that circleci config change with a single repo after we've copied the repos over to east-1 and east-2. The login/push pairing wasn't in place before, so I'm not sure if it will be necessary, but I'll keep that approach in mind while testing.

Reply:
The change looks to be working as expected. Pushing a change to a test branch  for `app-service`, the CI script for `dev-build-and-publish` completed, and pushed images to all 4 regions. [PR](https://github.com/Clever/app-service/pull/528) has been closed, and was only created to run CI scripts.
* [us-west-1](https://console.aws.amazon.com/ecr/repositories/private/589690932525/app-service/image/sha256:1e71809de0667df71b8c48e24569535adb9d4a45167256200cd5ce8167c04d55/details/?region=us-west-1)
* [us-west-2](https://console.aws.amazon.com/ecr/repositories/private/589690932525/app-service/image/sha256:1e71809de0667df71b8c48e24569535adb9d4a45167256200cd5ce8167c04d55/details/?region=us-west-2)
* [us-east-1](https://console.aws.amazon.com/ecr/repositories/private/589690932525/app-service/image/sha256:1e71809de0667df71b8c48e24569535adb9d4a45167256200cd5ce8167c04d55/details/?region=us-east-1)
* [us-east-2](https://console.aws.amazon.com/ecr/repositories/private/589690932525/app-service/image/sha256:1e71809de0667df71b8c48e24569535adb9d4a45167256200cd5ce8167c04d55/details/?region=us-east-2)